### DISCUSSION TITLE
Garmin 935

### DISCUSSION THREAD

**Discussion by urbanox** [maca88/SmartBikeLights#20] [General]
Great app, battery level is displayed but no light mode on Garmin 935. I have latest updates and see light mode in Garmin menu. Getting "Unknown light mode" icon.

**Comment by maca88** [comment]
You need to configure the lights that you have paired by using the [Lights Configurator](https://maca88.github.io/SmartBikeLights/), you can also check the "how to use" guide here: https://github.com/maca88/SmartBikeLights#how-to-use

**Reply by urbanox** [reply]
Thank you for your help, now it works. Looks great.
